.github/workflows/clean-artifacts.yml

```diff
@@ -0,0 +1,17 @@
+name: Remove old artifacts
+
+on:
+  schedule:
+    # Every day at 1am
```

Comment:
1am UTC?

Reply:
not sure the timezone but it doesnt matter, this is something that wont effect anything nor anyone will see it working